src/index.js

```diff
@@ -3,5 +3,11 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import 'bootstrap/dist/css/bootstrap.css';
+import { BrowserRouter as Router } from "react-router-dom";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+    <Router>
+    <App />
+    </Router>,
+    document.getElementById("root")
+    );
```

Comment:
Great job on this project. All features are implemented and working as expected. Only suggestion is code indenting some places to make it easier to read. For example, try to have <App /> indented. There are some VS Code extensions that will do this automatically for you when saving a file.